src/tests/testHttpRequest.cc

```diff
@@ -24,9 +24,6 @@ class TestHttpRequest : public CPPUNIT_NS::TestFixture
     CPPUNIT_TEST(testSanityCheckStartLine);
     CPPUNIT_TEST_SUITE_END();
 
-public:
-    void setUp() override;
```

Comment:
> PR description: CppUnit provides TestFixture::setUp() and tearDown() methods, but those methods are executed before and after _each_ test case.

In fact, the whole TestFixture object is created and destroyed for each test case, and the destruction waits for all test cases (within the same TestFixture class?) to complete. Here is the output from temporary branch debugging (removed in branch commit a3ab387) that illustrates that sequence of events:

```
$ src/tests/testHttpRequest
SKIP: mem/libmem.la MemPools (not implemented).
SKIP: security/libsecurity.la PeerOptions (not implemented).
SKIP: store/libstore.la Controller (not implemented).
XXX1b: Custom Before
SKIP: mem/libmem.la Init (not implemented).
XXX2: Default TestHttpRequest ctor
XXX2: Default TestHttpRequest ctor
XXX2: Default TestHttpRequest ctor
............
OK (12)
XXX8: Default TestHttpRequest dtor
XXX8: Default TestHttpRequest dtor
XXX8: Default TestHttpRequest dtor
XXX9: Default After
SKIP: store/libstore.la ~Controller (not implemented).
```